components/script/dom/htmlelement.rs

```diff
@@ -273,19 +302,80 @@ impl HTMLElementMethods for HTMLElement {
                 document.window().SetOnscroll(listener)
             }
         } else {
-            self.upcast::<EventTarget>().set_event_handler_common("scroll", listener)
+            self.upcast::<EventTarget>().set_event_handler_common(
+                "scroll",
+                listener,
+            )
+        }
+    }
+
+    // https://html.spec.whatwg.org/multipage/#attr-itemtype
+    fn Itemtypes(&self) -> Option<Vec<DOMString>> {
+        let item_attr: AttrValue = self.parse_plain_attribute(
+            &local_name!("itemtype"),
+            self.upcast::<Element>()
+                .GetAttribute(DOMString::from(String::from("itemtype")))
+                .unwrap_or(DOMString::from(String::from(""))),
+        );
```

Comment:
Rather than calling parse_plain_attribute here, we should use [Element::get_tokenlist_attribute](http://doc.servo.org/script/dom/element/struct.Element.html#method.get_tokenlist_attribute) instead, which will return the previously-parsed value.

Reply:
About this change,  do we then remove the parse_plain_attribute method that we implemented ? because its purpose was to be used in this function

Reply:
No, parse_plain_attribute is a method that gets called from other parts of Servo already. Your implementation is required, like adding a virtual method override in C++.

Reply:
Got it, we have now used Element::get_tokenlist_attribute and will push the code in few minutes, running test-tidy now. When can we expect the appveyor to work again? ;(

Reply:
AppVeyor passing is not required for landing. Also, when we finally approve your changes and are ready to land your changes, we have another build & test system that will build and test your changes against all web platform tests before merging your PR to the master branch.